script/install_boost.sh

```diff
@@ -120,16 +119,12 @@ then
     then
         ALPAKA_BOOST_B2_CXXFLAGS+=" -Wunused-private-field -Wno-unused-local-typedef -Wno-c99-extensions -Wno-variadic-macros"
     fi
-    # Select the libraries required.
-    # If the variable is not set, the backend will most probably be used by default so we install it.
-    if [ "${ALPAKA_CI_INSTALL_FIBERS}" == "ON" ]
```

Comment:
Why do you removed this check?

In cases where fiber is enabled we perform a real complex installation of boost. We do not want to have fiber installed in each check.

Reply:
IN cases we have not fiber enabled we do not need boost atomics, thraeds, ...

Reply:
Because the same check is already on line 69 and thus has no effect.